service/riotapi.py

```diff
@@ -50,7 +53,11 @@ def get_match_timeline(match_id, api_key):
 def get_response_json(url):
     response = requests.get(url)
     status_code = response.status_code
-    if (status_code != 200):
+    if status_code == 404:
         print(f"Status code: {status_code} for url: {url}")
-        exit(1)
-    return response.json()
\ No newline at end of file
+        return response.json()
```

Comment:
I think we could instead have the response json return 2 things at the same time, status code and the json. Then we can have methods check the code to see if they want to access the json? Does that solve the issue? Not sure where the error was originating...

Reply:
The error was in some way originating in the riotapi.py get_response_json (method < Terminology guess).

In the Main you say if not 200 Exit(1).
But by bypassing the Exit(1) the get_puuid (Method???)  on riotapi.py would fail. Since Key: puuid didn't exist for the return. 
and bypassing that would cause the second if statement to fail as len was expecting a type it was not seeing.

I would like to learn the right way to do this. Just saying, my way does solve.  XD